apps/src/lib/levelbuilder/course-editor/CourseEditor.js

```diff
@@ -332,6 +332,9 @@ class CourseEditor extends Component {
           handleParticipantAudienceChange={e =>
             this.setState({participantAudience: e.target.value})
           }
+          cannotChangeParticipantType={
```

Comment:
naming gripe: in general and for next time, I would suggest positive variable names e.g. `canChangeParticipantType`. this helps avoid situations where you would have to use double negatives like `!cannotChangeParticipantType`. or if you want to avoid inverting all the logic you've already written, it could be `preventChangeParticipantType`.

Reply:
Updated!